src/util/calculate.ts

```diff
@@ -21,3 +21,12 @@ export const getValueInPercentage = (
 export const roundWithOneDecimal = (value: number): number => {
   return Math.round(value * 10) / 10;
 };
+
+export const bytesToString = (value = 0, decimals = 2): string => {
+    if (value === 0) return '0 Bytes';
+    const k = 1024;
+    decimals = decimals < 0 ? 0 : decimals;
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+    const i = Math.floor(Math.log(value) / Math.log(k));
+    return `${parseFloat((value / k ** i).toFixed(decimals))} ${sizes[i]}`;
+}
```

Comment:
Let's move this to its own file

Reply:
Generic file to be added to later like `src/util/convert.ts` or just it's own `src/util/bytes-to-string.ts`?

Reply:
it's own